elastic-mapping-updater/src/test/resources/template1.json

```diff
@@ -1,184 +1,194 @@
+                        "type": "pattern_capture",
+                        "preserve_original": false,
+                        "patterns": [
+                            "^([\\\\p{Nd},\\\\.'?]+)$|([^\\\\.']+)"
+                        ],
+                        "consume_all_tokens": false
                     },
-                    "MESSAGE": {
-                        "type": "text",
-                        "copy_to": [
-                            "FAILURES.MESSAGE_AGGREGATE"
+                    "max_token_length_filter": {
+                        "type": "length",
+                        "min": 0,
+                        "max": 10922,
+                        "consume_all_tokens": false
+                    }
+                },
+                "analyzer": {
+                    "general_analyzer": {
+                        "type": "custom",
+                        "filter": [
+                            "million_token_limit",
+                            "max_token_length_filter",
+                            "icu_normalizer"
                         ],
-                        "analyzer": "general_analyzer"
+                        "char_filter": [],
+                        "tokenizer": "icu_tokenizer"
                     },
-                    "MESSAGE_AGGREGATE": {
-                        "type": "keyword",
-                        "ignore_above": 256
+                    "path_analyzer": {
+                        "type": "custom",
+                        "tokenizer": "path_analyzer",
+                        "filter": [
+                            "max_token_length_filter",
+                            "icu_normalizer"
+                        ]
                     },
-                    "WORKFLOW_ACTION": {
-                        "type": "keyword",
-                        "ignore_above": 32
+                    "version_analyzer": {
+                        "type": "pattern",
+                        "pattern": "\\\\s+"
                     },
-                    "WORKFLOW_NAME": {
-                        "type": "keyword",
-                        "ignore_above": 100
+                    "standard_max_term_100": {
+                        "type": "standard",
+                        "stopwords": "_english_",
+                        "max_token_length": 100
+                    }
+                },
+                "normalizer": {
+                    "keyword_normalizer": {
+                        "type": "custom",
+                        "filter": [
+                            "icu_normalizer"
+                        ]
+                    }
+                },
+                "tokenizer": {
+                    "path_analyzer": {
+                        "type": "path_hierarchy",
+                        "delimiter": "/",
+                        "replacement": "/",
+                        "buffer_size": 1024,
+                        "reverse": false,
+                        "skip": 0
```

Comment:
Had to add in default value for all fields in `path hierarchy tokenizer` despite them being optional. In the java client they are required in order for the index template request to be built. There are changes like this in all of the templates

Reply:
Resolved by the changes to perform DANGEROUS deserialization.